lib/Doctrine/Common/Annotations/Annotation/Target.php

```diff
@@ -25,14 +25,16 @@ final class Target
     public const TARGET_METHOD     = 2;
     public const TARGET_PROPERTY   = 4;
     public const TARGET_ANNOTATION = 8;
-    public const TARGET_ALL        = 15;
+    public const TARGET_FUNCTION   = 16;
+    public const TARGET_ALL        = 31;
```

Comment:
That's a BC-break isn't it?

Reply:
I'm not sure here, these constants are meant to be used as bitmasks and since the new value includes all the bits of the old value it doesn't break BC.

Of course if someone is doing something strange with this value (like comparing directly with 15 instead of using it as a bitmask) it could leads to BC breaks but I don't see a good reason for someone to do that.

Reply:
Sounds fair enough